### PR TITLE
Rephrase when `send` blocks

### PR DESCRIPTION
**Pull request by mgeisler**
From a discussion with @hueich: https://github.com/google/comprehensive-rust/pull/786#discussion_r1224850499.

### CODE REVIEW

**Comment by mgeisler** [comment]
Hi @hueich, I hope this makes the sentence a little clearer.

**Review by hueich — APPROVED** [review]
Looks clearer, thanks!

**Comment by hueich** [comment]
Btw, is it possible to create a zero-length bounded channel (like in Go)?  If so, what's the behavior there?

**Comment by mgeisler** [comment]
> Btw, is it possible to create a zero-length bounded channel (like in Go)? If so, what's the behavior there?

Yes, that's possible and while reading up on channels, I learned that this is called a "rendezvous channel". Every `send` will block until a `read` is done. I can add a line about this.

File: src/concurrency/channels/bounded.md [hunk 1]
```diff
@@ -25,3 +25,10 @@ fn main() {
     }
 }
 ```
+
+<details>
+    
+* Calling `send` will block the currnet thread until there is space in the channel for the new message. The thread can be blocked indefinitely if there is nobody who reads from the channel.
+* A call to `send` will abort with an error (that is why it returns `Result`) if the channel is closed. A channel is closed when the receiver is dropped.
```
Comment (mgeisler) [inline src/concurrency/channels/bounded.md:32]:
```suggestion
* A call to `send` will abort with an error (that is why it returns `Result`) if the channel is closed. A channel is closed when the receiver is dropped.
* A bounded channel with a size of zero is called a "rendezvous channel". Every send will block the current thread until another thread calls `read`.
```

**Pull request merged**